### PR TITLE
Telemetry data eda

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

#29 #33 

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

Implements EDA for **telemetry** dataset. Implements: 

- how to load data into notebook 
- provides high level analysis
- discusses differences between BackBlaze and telemetry datasets

### CODE REVIEW

**Comment by MichaelClifford** [comment]
@isabelizimm does this PR require a rebase? It looks like it has conflicts with 4 other notebooks?

**Comment by chauhankaranraj** [comment]
@isabelizimm since we're reading the data directly from the MOC s3, we do need to add the .zip and the .csv files here? I'd suggest removing them so that there's no redundancy. wdyt?

**Comment by isabelizimm** [comment]
> @isabelizimm since we're reading the data directly from the MOC s3, we do need to add the .zip and the .csv files here? I'd suggest removing them so that there's no redundancy. wdyt?

I can remove those! They will be created locally whenever the `wget` and `ZipFile` commands are run, but no need to have on GitHub

**Comment by chauhankaranraj** [comment]
> I can remove those! They will be created locally whenever the `wget` and `ZipFile` commands are run, but no need to have on GitHub.

Sounds good, thanks!

> However, we do have a directory `./data/` to have the csvs at hand. Do we remove that as well?

Good question, I think ideally we should add the following directories with a `.gitkeep` in them, to make this repo more consistent with the [project-template](https://github.com/aicoe-aiops/project-template).
```
.
└── data
    ├── external
    │   └── .gitkeep
    ├── interim
    │   └── .gitkeep
    ├── processed
    │   └── .gitkeep
    └── raw
        └── .gitkeep
```
But I don't wanna block this PR, so maybe for now we can just remove the `./data` dir from this PR, and update the data directories structure in a separate PR? wdyt?

**Comment by isabelizimm** [comment]
@chauhankaranraj Agreed-- and was already on removing that directory. :) I like the idea of a .gitkeep, lets add that in a different issue.

**Comment by chauhankaranraj** [comment]
> @chauhankaranraj Agreed-- and was already on removing that directory. :) I like the idea of a .gitkeep, lets add that in a different issue.

Created #40 to address this

**Comment by chauhankaranraj** [comment]
> @isabelizimm does this PR require a rebase? It looks like it has conflicts with 4 other notebooks?

@MichaelClifford I believe this PR is ready for review now :)

File: notebooks/data_sources/telemetry/step0_EDA.ipynb [hunk 1]
```diff
@@ -0,0 +1,1091 @@
+{
```
Comment (MichaelClifford) [inline notebooks/data_sources/telemetry/step0_EDA.ipynb:1]: <p>Do you want to list the available files from the swift endpoint (or a partial list MAX of 5) so people can see whats available and what the naming convention is?</p><p> </p><p>Should this be <code>month</code>  or <code>file_name</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ceph_drive_failure/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Pull request merged**